Cargo.lock

```diff
@@ -1545,10 +1567,24 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "9c972679f83bdf9c42bd905396b6c3588a843a17f0f16dfcfa3e2c5d57441835"
 dependencies = [
  "darling_core",
- "quote 1.0.33",
+ "quote 1.0.37",
  "syn 1.0.95",
 ]
 
+[[package]]
+name = "dashmap"
```

Comment:
didn't we remove this from the walletconnect repo PR, we should remove it here too.

Reply:
removed https://github.com/KomodoPlatform/komodo-defi-framework/pull/2223/commits/4e82c4fad3a264aa8fe5a781137655e92e48c1d8